dev

```diff
@@ -56,7 +57,8 @@ def depend_on(binaries=[], envs=[]):
     return decorator_depend_on
 
 
-@depend_on(binaries=["kubectl", "kind", "docker"], envs=["Z2JH_KUBE_VERSION"])
+@depend_on(binaries=["kubectl", "kind", "docker"],
+    envs=["Z2JH_KUBE_VERSION", "Z2JH_HELM_VERSION"])
```

Comment:
We should probably not specify this here as we don't need it to have a value in the script. It made sense for me to introduce in order to enforce specifications of various variables.

Hmm... Thinking about it, if Z2JH_HELM_VERSION is omitted, we will use the currently available helm CLI, which could be v2 or v3. It would make more sense that the logic below checked for the helm version using the helm cli to determine how to act instead of probing this variable.

Reply:
I'm a bit worried about autodetecting config in CI, because if someone makes an incorrect change to any of the CI scripts or Travis config the auto-configuration could skip over it and the tests would appear to pass, when in fact a portion of them might not have been run.

It could be made opt-in (add another flag e.g. `dev --autodetect-env`?) though that's still not very user friendly.

Reply:
Okay, we don't need to autodetect, but I think we should not require the explicit specification of the helm version using the depend_on decorator that will make the function fail unless it is specified. We are looking at the env var with a default of a blank string later anyhow.

So, not using the `helm version` to get the version of helm, but also not requiring the Z2JH_HELM_VERSION to be specified, makes sense to me short term. I want to revise this dev script again as soon as I find time btw. It is not good enough for local development :(

Reply:
I understand now, done!